### ISSUE TITLE
Provide a way to get preview and tracks with one request

### ISSUE THREAD

**Issue by D3SOX** [microlinkhq/spotify-url-info#90]
Currently, if I want to get both the preview and the tracks I have to call both `getPreview` and `getTracks` which unnecessarily makes two requests. It would be nice if there was a method that returns both like
```ts
getDetails(url: string, options?: object): Promise<{ preview: Preview, tracks: Tracks[] }>;
```

**Comment by Kikobeats** [comment]
I like the idea, and it's easy to achieve; Appreciated if you can make a PR 🙂

**Comment by D3SOX** [comment]
Okay, I can look into this later

**Comment by D3SOX** [comment]
Not related to this, but @Kikobeats Why are lock files ignored in this repository?
[Why you should commit a lock file](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json)

**Comment by Kikobeats** [comment]
Released at spotify-url-info@3.1.0 🎉

**Comment by D3SOX** [comment]
Thank you! I saw the coverage decreased, but I don't think a test for this is necessary because it's actually already covered by the other tests.
